lib/pos_mempool.go

```diff
@@ -672,6 +851,15 @@ func (mp *PosMempool) GetAugmentedUniversalView() (*UtxoView, error) {
 func (mp *PosMempool) GetAugmentedUtxoViewForPublicKey(pk []byte, optionalTx *MsgDeSoTxn) (*UtxoView, error) {
 	return mp.GetAugmentedUniversalView()
 }
+func (mp *PosMempool) BlockUntilReadOnlyViewRegenerated() {
+	oldSeqNum := atomic.LoadInt64(&mp.augmentedLatestBlockViewSequenceNumber)
+	newSeqNum := oldSeqNum
+	for newSeqNum == oldSeqNum {
+		// Check fairly often. Not too often.
+		time.Sleep(25 * time.Millisecond)
```

Comment:
For symmetry, I'd make this `augmentedBlockViewRefreshIntervalMillis / 10` so that it checks faster if you update that variable.

Reply:
Update: Resolved this myself.